playbooks/roles/foreman_repositories/tasks/main.yml

```diff
@@ -1,6 +1,9 @@
 ---
-- include: release_repos.yml
-  when: foreman_repositories_use_release and not foreman_repositories_use_koji
+- include: debian_release_repos.yml
```

Comment:
I like it, as it removes quite some "skipped" lines (depending on the Ansible version) from the output.